src/components/ModalWindow/AddModal/AddModal.tsx

```diff
@@ -0,0 +1,175 @@
+import React, {SyntheticEvent} from "react";
+import {
+  Form,
+  TitleModal,
+  InputDate,
+  InputLeft,
+  InputRight,
+  Label,
+  StyledFlex,
+  StyledFlexForButtons,
+  TextArea
+} from "./AddModal.styled";
+import {PrimaryButton, SecondaryButton} from "../../Button/Button.styled";
+import {genreOptions} from "../../../assets/data/constData";
+import {backgroundGrey, backgroundMain, baseWeight, mainFontColor, xlarge, red} from "../../../styles/global_varables";
+import {Multiselect} from "multiselect-react-dropdown";
+
+interface ModalWindowProps {
+  form: {
```

Comment:
This should be of type of `MovieFormProps` that was discussed above, thats' the entire point of the interface